source/user-manual/ruleset/custom.rst

```diff
@@ -63,23 +62,23 @@ Check out this example on how to create new decoders and rules. The following lo
       Dec 25 20:45:02 MyHost example[12345]: User 'admin' logged from '192.168.1.100'
 
       **Phase 1: Completed pre-decoding.
```

Comment:
Verify the`wazuh-logtest` output indexation. Here's screenshot where you can see that the text for each phase starts just below each phase number: 1, 2, 3.

![image](https://github.com/wazuh/wazuh-documentation/assets/61882981/52753806-0c1a-4977-9a3d-8244fe986ca8)